tests/resources/proton_run000201_za20deg_azm000deg_North_alpha_6.0.0_test_file/config.yml

```diff
@@ -0,0 +1,26 @@
+---
+CTA_SIMPIPE:
+  APPLICATIONS:
+  - APPLICATION: simtools-simulate-prod
+    CONFIGURATION:
+      SIMULATION_SOFTWARE: corsika_sim_telarray
+      LABEL: test_file
+      MODEL_VERSION: 6.0.0
+      SITE: North
+      ARRAY_LAYOUT_NAME: alpha
+      PRIMARY: proton
+      RUN_NUMBER: 201
+      AZIMUTH_ANGLE: 0
+      ZENITH_ANGLE: 20
+      NSHOW: 50
+      VIEW_CONE: 0 deg 10 deg
+      ENERGY_RANGE: 30 GeV 600000 GeV
+      CORE_SCATTER: 20 2000 m
```

Comment:
2000 m would be a bit too much, no? We use 1900 m for the hyper array in Prod6.

Reply:
Again - if we get triggers, does it matter? Also we want to use for the limit derivation a very wide definition. (but I agree with you that 2000 m are a lot)

Reply:
Just to avoid waste and having this example there, I would change it, but won't complain if you decide to keep.